x-team/auto-ui
# Add play/pause SVG icons

**Release Type:** *New feature* 


Fixes x-team/auto#1168

## Description

- Adds the play/pause SVG icons.

## Checklist



## Related PRs

branch | PR
------ | ------
`1168-avatar-module-visual-styling` | [Auto](https://github.com/x-team/auto/pull/1272)

## Steps to Test or Reproduce

- Navigate to the `SvgIcon` component section.
- Make sure the new icons are shown.
